app/bundles/AssetBundle/Translations/en_US/messages.ini

```diff
@@ -19,7 +19,7 @@ mautic.asset.asset.form.storageLocation.local="Local"
 mautic.asset.asset.form.storageLocation.remote="Remote"
 mautic.asset.asset.form.disallow.crawlers="Block search engines from indexing this file"
 mautic.asset.asset.form.disallow.crawlers.descr="If you don't want to index files like PDF, DOCX etc, this option will disallow search bots by using the X-Robots-Tag HTTP header."
-mautic.asset.asset.help.alias="Letters and numbers (hyphens allowed) used for URL generation of this asset. A unique alias based on the title will be autogenerated if left empty."
+mautic.asset.asset.help.alias="Letters and numbers (hyphens allowed) are used for URL generation of this asset. A unique alias based on the title will be autogenerated if left empty."
```

Comment:
This one won't make sense in the context where it's used

Reply:
reverted